common/big_messaging/src/test/scala/uk/ac/wellcome/big_messaging/fixtures/BigMessaging.scala

```diff
@@ -0,0 +1,136 @@
+package uk.ac.wellcome.bigmessaging.fixtures
+
+import akka.actor.ActorSystem
+import com.amazonaws.services.cloudwatch.model.StandardUnit
+import com.amazonaws.services.sqs.model.SendMessageResult
+import com.amazonaws.services.sns.AmazonSNS
+import io.circe.{Decoder, Encoder}
+import org.scalatest.Matchers
+import uk.ac.wellcome.akka.fixtures.Akka
+import uk.ac.wellcome.bigmessaging.message._
+import uk.ac.wellcome.fixtures.TestWith
+import uk.ac.wellcome.json.JsonUtil._
+import uk.ac.wellcome.messaging.MessageSender
+import uk.ac.wellcome.messaging.sns.{SNSConfig, SNSMessageSender}
+import uk.ac.wellcome.messaging.memory.MemoryMessageSender
+import uk.ac.wellcome.messaging.fixtures.{SNS, SQS}
+import uk.ac.wellcome.messaging.fixtures.SNS.Topic
+import uk.ac.wellcome.messaging.fixtures.SQS.Queue
+import uk.ac.wellcome.monitoring.memory.MemoryMetrics
+import uk.ac.wellcome.storage.{ObjectStore, StorageBackend}
+import uk.ac.wellcome.storage.fixtures.S3
+import uk.ac.wellcome.monitoring.fixtures.MetricsSenderFixture
+import uk.ac.wellcome.bigmessaging.BigMessageSender
+import uk.ac.wellcome.storage.fixtures.S3.Bucket
+import uk.ac.wellcome.storage.streaming.Codec
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.util.Success
+
+trait BigMessaging
```

Comment:
We tend to append Fixture to the name (i.e. `BigMessagingFixture`) I know the namespace indicates it but i've found it helps with clarity.